src/components/foodDescription/FoodDescription.js

```diff
@@ -0,0 +1,25 @@
+// lodash
+import _keys from 'lodash/keys'
+
+// components
+import Sidebar from "./components/sidebar";
+import FoodList from './components/foodList';
+
+// css
+import './foodDescription.css'
+
+
+function FoodDescription(props) {
+    
+    const { foodListData } = props;
+
+    return (
+        <div className="flex food-description">
+            <Sidebar categories={_keys(foodListData)} />
+            <FoodList foodListData={foodListData} />
```

Comment:
foodList is enough, no need to add Data as suffix

Reply:
no, here foodListData refers to an object having array of foodItems. so the inside array will be food list
ex foodListData = { breakfast:[], starters:[]}

Reply:
> foodList is enough, no need to add Data as suffix

++

Reply:
> 

then may be you can call it foodItems, so items can have category as well the list of product inside the category